### DISCUSSION TITLE
`COPY INTO <location>` to have output reflecting the export results

### DISCUSSION THREAD

**Discussion by soyeric128** [databendlabs/databend#13164] [Ideas]
`COPY INTO <table>` now provides an output containing the import details: https://databend.rs/doc/sql-commands/dml/dml-copy-into-table#output

It's nice for `COPY INTO <location>` to have a similar output reflecting the export results, which includes the file names and other details like this:

![20231010-135102](https://github.com/datafuselabs/databend/assets/106025534/b8da6065-c0b8-4300-8e71-4087f1512e4e)

**Comment by wubx** [comment]
copy  into table  and copy into location user care about different. 
Copy into table is load data into table: care about which file and many rows load, because it's allow skip file. 
copy into location is one transaction action just need it work ok.
